packages/react/src/components/OverflowMenu/next/OverflowMenu.stories.js

```diff
@@ -26,11 +25,6 @@ export default {
   subcomponents: {
     OverflowMenuItem,
   },
-  parameters: {
-    docs: {
-      page: mdx,
-    },
-  },
 };
```

Comment:
This MDX page seems like it could be helpful, was it causing problems when you were bringing it in?